data/mock-12/README.md

```diff
@@ -0,0 +1,11 @@
+#mock-12
+
+Composed of 27 bacterial strains amplified with 515f/806r primers. Intended to include more fine-scale variation than the other mock communities, the members of which were chosen in part for their well-separated 16S rRNA gene sequences. The Extreme strains are all distinguishable over the sequenced region of the 16S rRNA gene, but some pairs of strains differ by as little as one nucleotide. Generated by Benjamin Callahan at Stanford University in 2015.
```

Comment:
*The Extreme strains* -> *These strains*, because you haven't mentioned the name *extreme* yet.